web/views.py

```
@@ -305,6 +310,160 @@ def index(request):
+        messages.success(request, "Survey submitted successfully!")
+        return redirect("survey-results", pk=survey.id)
+
+    return redirect("survey-detail", pk=survey.id)
+
+
+class SurveyResultsView(LoginRequiredMixin, DetailView):
+    model = Survey
+    template_name = "surveys/results.html"
+    login_url = "/accounts/login/"
+
+    def get_context_data(self, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_context_data`

(ANN201)